CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0 
+- update mongodb version to 3.6.4 
```

Comment:
I think it was done in the previous version. Here we just updated @types/mongodb package.

Reply:
It's only upgrading the devDependencies that are used only by this repo, I don't feel these changes are really necessary. The text your';e quoting from me was about common-lambda, not this repository.

Reply:
hmmm so we do not also need to update this repository to ensure the fix is propagated forward?

Reply:
You don't need to update this repository, mongodb is a peer dependency, it means it's going to use the mongodb package used by the repo where mongo-cursor-pagination-alt is added as a dependency. The work is already done, might as well merge it but it's not gonna do anything at the common-lambda level.